cmd/beacon-chain/storage/options.go

```diff
@@ -24,8 +25,28 @@ var (
 		Value:   uint64(params.BeaconConfig().MinEpochsForBlobsSidecarsRequest),
 		Aliases: []string{"extend-blob-retention-epoch"},
 	}
+	BlobStorageLayout = &cli.StringFlag{
+		Name:   "blob-storage-layout",
+		Usage:  layoutFlagUsage(),
+		Value:  filesystem.LayoutNameFlat,
+		Action: validateLayoutFlag,
+	}
 )
 
+func layoutFlagUsage() string {
+	return "Dictates how to organize the blob directory structure on disk. " +
+		"Available options are: " + strings.Join(filesystem.LayoutNames, ", ")
+}
+
+func validateLayoutFlag(_ *cli.Context, v string) error {
+	for _, l := range filesystem.LayoutNames {
```

Comment:
Why not using a set (`map[string]bool`) instead?

It could avoid the usage of the  `O(n)` range/if and replace it by a single `O(1)` operation.
(I agree here we won't gain much time since there is 2 items in the list, but in general `O(1)` is better than `O(n)`).

Reply:
I considered this, but there are only 2 values, and this is just used once at option processing, so I thought readability for the win here - plus I suspect iterating over a 2 item list may be faster than a map lookup (especially in all the cases where the first item is a match haha).

Reply:
(oh yeah and the list let me use `strings.Join` in the help text...)